'''a=10
b=0
c=a/b  # exception raised
print(c)'''
'''
a=10
b=0
try:
    c=a/b #you write exception in try block
    print(c)
except:
    print('koti') # solution in except block'''

